### ISSUE TITLE
Default params with plain `use Params.Schema`

### ISSUE THREAD

**Issue by astery** [vic/params#14]
Hello, here is my issue:

```elixir
defmodule MyParams do
  use Params.Schema

  schema do
    field :count, :integer, default: 1
  end
end

changeset = MyParams.from(%{})
assert %{count: 1} = Params.to_map(changeset) # but it is %{}
```

**Comment by vic** [comment]
Hm, this is a bug, I guess we need a new test for it the latest PR to work on default vales was #10

**Comment by vic** [comment]
oh btw @lasseebert, any idea about this issue?

**Comment by lasseebert** [comment]
This is definitely a bug. I will have a look when time allows ;)

**Comment by vic** [comment]
closing, as it has been merged in master and published as 2.0.6
